packages/office-ui-fabric-react/src/common/_themeVariables.scss

```diff
@@ -0,0 +1,55 @@
+$ms-font-tiny-fontFamily: "[theme:ms-font-tiny-fontFamily, default: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif]";
```

Comment:
need to export this file like the rest (or make one megafile that imports each of these and just export that conglomerate file, like we discussed)